src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -763,6 +766,40 @@ public class SimpleEvents {
 				.since("INSERT VERSION");
 		}
 
+		if (Skript.classExists("io.papermc.paper.event.world.border.WorldBorderEvent")) {
+			Skript.registerEvent("World Border Bounds Change", SimpleEvent.class, WorldBorderBoundsChangeEvent.class, "[world[ ]]border [bounds] chang(e|ing)")
+				.description(
+					"Called when a world border changes its bounds, either over time, or instantly.",
+					"Note: event-number will be the diameter of the world border."
+				)
+				.requiredPlugins("Paper 1.16+")
```

Comment:
```suggestion
				.requiredPlugins("Paper")
```

Reply:
In the past I have heard that it is a good idea to leave the version anyway even if it is impossible to not use that with the current release of skript (that was worded weirdly). Is this policy changing?

Reply:
no idea tbh